department-of-veterans-affairs/abd-vro
# Disable Service Ruby

# Disable Service Ruby 

## Description



### What was the problem?

When docker compose starts all the services, `service-ruby` would conflict with `service-python` when connecting to RabbitMQ.

### How does this fix it?

This removes `service-ruby` from the build and docker steps but keeps the files for reference. If we ever need to go back, a PR revert will suffice.

## How to test this PR

- When you run `./gradlew :app:dockerComposeDown :app:dcPrune :app:dockerComposeUp`, check that a `service-ruby container has not been created/started
